controllers/managedmcg_controller.go

```diff
@@ -436,6 +438,22 @@ func (r *ManagedMCGReconciler) removeNoobaa() error {
 			return fmt.Errorf("failed to delete bucketClasses: %w", err)
 		}
 	}
+	timeout := 10 * time.Second
```

Comment:
Is 10 secs enough ?

Reply:
yes @v-harihar, most of the time condition will be true within first iteration